tetrateci/create_istio_release.sh

```diff
@@ -106,8 +106,7 @@ sudo rm -rf /tmp/istio-release/sources/
 # If RELEASE, Build Archives
 if [[ -z ${TEST:-} ]]; then
     echo "Building archives..."
-    mkdir /tmp/istio-release
```

Comment:
can you help me reminding why we introduced `mkdir /tmp/istio-release` before? maybe share some previous PR?
Just want to ensure removal does not introduce some unintended behavior. and indeed it can be safely removed.

Reply:
mkdir was already there. Prior to 1.13, once docker images are built, we cleanup the folder and create a new folder and start archiving.

Reply:
do you mean in line 75, https://github.com/tetratelabs/istio/pull/743/files#diff-0c53a24edec946a54ee2c4fc09f164520a71d0a676ad949a8a07de4ced80921aR75

the dir has already been created? if so, recreating it here will make the CI fails?

Reply:
yes, 
go run main.go build --manifest manifest.docker.yaml - this creates artifacts in /tmp/istio-release
then
go run main.go build --manifest manifest.archive.yaml - does the same, so it needs clean folder to build everything here.